Site/ASAB Maestro/Files/asab-remote-control/asab-config/General/__schema__.json

```diff
@@ -35,7 +35,71 @@
 					]
 				}
 			}
-		}
+		},
+		"smtp": {
+			"type": "object",
+			"title": "SMTP Server",
+			"description": "Add configuration for the SMTP server to send emails from the system.",
+			"properties": {
+				"host": {
+					"type": "string",
+					"title": "Host",
+					"description": "Where is your SMTP server.",
+					"examples": "https://smtp-provider.com/"
+					},
+				"user": {
+					"type": "string",
+					"title": "User",
+					"description": "Set credentials for the SMTP server. Type 'apikey' and fill in apikey into password option if needed."
+					},
+				"password": {
+					"type": "string",
+					"title": "Password",
+					"description": "Set credentials for the SMTP server."
+					},
+				"from": {
+					"type": "string",
+					"format": "email",
+					"title": "From",
+					"description": "Set default sender email address for this system."
+					},
+				"subject": {
+					"type": "string",
+					"title": "Subject",
+					"description": "Set default email subject for this system."
+					},
+				"ssl": {
+					"type": "string",
+					"enum": ["yes", "no"],
+					"title": "SSL",
+					"description": "Switch SSL on?",
+					"default": [
+						"yes",
+						"no"
+					],
+					"$defs": {
+						"select": {
+							"type": "select"
+							}
+						}
+					},
```

Comment:
UI should render this as a "Checkbox" @Pe5h4

Reply:
Then the label is "SSL/TLS" only.

Reply:
Comment updated

Reply:
jak to může být checkbox, když je to výběr ze dvou možností? 
Switch between SSL and TLS -> check! a vybrala jsem co?

Nevím ani, jak je to naimplementované tahle konfigurace...

@Pe5h4 jak udělám, aby ten checkbox byl zaškrtnutý, když default je true?

Reply:
když tu konfiguraci na UI vyplním špatně, protože nepasuje do toho schémata, tak mi všechno, co už jsem vyplnila, zmizí a neřekne mi to, který to políčko je špatně :( 
Říkat, které políčko je špatně bude muset BE, ale kdyby to na FE nemizelo... prosím prosím...

Reply:
Obrázek odpovídá konifguraci.
Nevím, jestli si s tím poradí BE - měl by.

![image](https://github.com/TeskaLabs/asab-maestro-library/assets/82450969/a6659582-3d56-4c30-a918-23ee7f9af53b)

```
{
    "library": {
        "providers": [ ]
    },
    "smtp": {
        "host": "https://eliska.cz",
        "user": "eliska",
        "password": "eliska",
        "from": "eliska@eliska.com",
        "subject": "eliska",
        "ssl": true,
        "starttls": false
    }
}
```